code/optional/solution/optional.sol.cpp

```diff
@@ -0,0 +1,28 @@
+#include <cmath>
+#include <iostream>
+#include <optional>
+
+std::optional<double> mysqrt( double d )
+ {
+  if (d<0) { return std::nullopt ; }
```

Comment:
```suggestion
  if (d<0) { return {}; }
```

Reply:
I usually also prefer `{}`, but `std::nullopt` is also fine.